custom_components/tahoma/climate_devices/atlantic_heat_recovery_ventilation.py

```diff
@@ -0,0 +1,257 @@
+"""Support for AtlanticHeatRecoveryVentilation."""
+
+import logging
+import time
+from datetime import date
+from typing import List, Optional
+
+from homeassistant.components.climate import ClimateEntity
+from homeassistant.components.climate.const import (
+    FAN_AUTO,
+    HVAC_MODE_FAN_ONLY,
+    SUPPORT_FAN_MODE,
+    SUPPORT_PRESET_MODE,
+)
+from homeassistant.const import EVENT_HOMEASSISTANT_START, STATE_UNKNOWN, TEMP_CELSIUS
+from homeassistant.core import callback
+from homeassistant.helpers.event import async_track_state_change
+
+from ..coordinator import TahomaDataUpdateCoordinator
+from ..tahoma_entity import TahomaEntity
+
+FAN_BOOST = "Home Boost"
+FAN_KITCHEN = "Kitchen Boost"
+FAN_AWAY = "Away"
+FAN_BYPASS = "Bypass Boost"
+
+PRESET_AUTO = "Auto"
+PRESET_PROG = "Prog"
+PRESET_MANUAL = "Manual"
+
+COMMAND_SET_AIR_DEMAND_MODE = "setAirDemandMode"
+COMMAND_SET_VENTILATION_CONFIGURATION_MODE = "setVentilationConfigurationMode"
+COMMAND_SET_VENTILATION_MODE = "setVentilationMode"
+COMMAND_REFRESH_VENTILATION_STATE = "refreshVentilationState"
+COMMAND_REFRESH_VENTILATION_CONFIGURATION_MODE = "refreshVentilationConfigurationMode"
+
+IO_AIR_DEMAND_MODE_STATE = "io:AirDemandModeState"
+
+FAN_MODE_TO_TAHOMA = {
+    FAN_AUTO: "auto",
+    FAN_BOOST: "boost",
+    FAN_KITCHEN: "high",
+    FAN_AWAY: "away",
+}
+
+TAHOMA_TO_FAN_MODE = {v: k for k, v in FAN_MODE_TO_TAHOMA.items()}
+
+HVAC_MODES = [HVAC_MODE_FAN_ONLY]
+PRESET_MODES = [PRESET_AUTO, PRESET_PROG, PRESET_MANUAL]
+FAN_MODES = [FAN_AUTO, FAN_BOOST, FAN_KITCHEN, FAN_AWAY, FAN_BYPASS]
+
+date=date.today()
+DAY=date.day
+MONTH=date.month
+
+VENTILATION_MODE_STATE = {"dayNight": "night","month": MONTH,"test": "off","endOfLineTest": "off","leapYear": "off","day": DAY,"cooling": "off","prog": "off"}
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class AtlanticHeatRecoveryVentilation(TahomaEntity, ClimateEntity):
+    """Representation of a AtlanticHeatRecoveryVentilation device."""
+
+    def __init__(self, device_url: str, coordinator: TahomaDataUpdateCoordinator):
```

Comment:
@tetienne could you perhaps help me with abstracting the logic for the retrieval of an external temperature sensor? I couldn't get it working. Important is that we can select the number (#4) for the temperature sensor.

Reply:
@iMicknl : What is the problem? It seems to work fine in my case.
![image](https://user-images.githubusercontent.com/11087736/119991967-524cdc80-bfca-11eb-9e06-9d77b4ab7eb8.png)
![image](https://user-images.githubusercontent.com/11087736/119992060-701a4180-bfca-11eb-9971-3cf2ffa4625b.png)

Reply:
Ok, I understand, thanks